File: files/loop.java

class loop{
    public static void main(String[] args) {
        int x = 0;
        for (;x < 10;) {
            x = x + 1;
        };
        int i = 0;
        for (; i < 10; i += 1) {
            System.out.println(i);
        }
        for (;;) {
            System.out.println("Infinite loop");
        }
    }
}

File: files/loop.go
package main

import "fmt"

func main() {
	var x int = 0
	for x < 10 {
		x = x + 1
	}
	var i int = 0
	for ; i < 10; i += 1 {
		fmt.Println(i)
	}
	for {
		fmt.Println("Infinite loop")
	}
}
